src/gdpr/Tracker.js

```diff
@@ -28,11 +29,12 @@ function getPvUniqueId() {
 }
 
 class Tracker {
-    constructor(language, detectedGeo, beaconCookieName, enable) {
+    constructor(language, detectedGeo, detectedRegion, beaconCookieName, enable) {
         this.enable = enable;
         this.defaultParams = {
             [TRACK_PARAMS.LANGUAGE_CODE]: language,
             [TRACK_PARAMS.DETECTED_GEO]: detectedGeo,
```

Comment:
it will still report country & region in lower case. right?
```suggestion
            [TRACK_PARAMS.DETECTED_GEO]: detectedGeo.toUpperCase(),
```